src/Compilers/CSharp/Portable/Syntax/IndexerDeclarationSyntax.cs

```diff
@@ -37,9 +37,9 @@ public partial class SyntaxFactory
             SyntaxList<AttributeListSyntax> attributeLists,
             SyntaxTokenList modifiers,
             TypeSyntax type,
-            ExplicitInterfaceSpecifierSyntax explicitInterfaceSpecifier,
+            ExplicitInterfaceSpecifierSyntax? explicitInterfaceSpecifier,
             BracketedParameterListSyntax parameterList,
-            AccessorListSyntax accessorList)
+            AccessorListSyntax? accessorList)
```

Comment:
Same comment.

Reply:
Not sure what the scenario is but IndexerDeclarationSyntax allows null accessor list and the IDE has explicitly passes null here:https://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp.Workspaces/CodeGeneration/PropertyGenerator.cs,265